ddtrace/internal/agent.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from ddtrace import compat
+
+from .uds import UDSHTTPConnection
+
+
+HOSTNAME = os.environ.get("DD_AGENT_HOST", os.environ.get("DATADOG_TRACE_AGENT_HOSTNAME", "localhost"))
+PORT = int(os.environ.get("DD_AGENT_PORT", os.environ.get("DD_TRACE_AGENT_PORT", 8126)))
+URL = os.environ.get("DD_TRACE_AGENT_URL", "http://%s:%d" % (HOSTNAME, PORT))
```

Comment:
Following the approach in #2041, it'd be great to have the environment lookup done during worker initialization.